cfe_internal/enterprise/federation/federation.cf

```diff
@@ -24,19 +24,31 @@ bundle agent config
         int => "20",
         comment => "Dump data on the feeders every 20 minutes";
 
+      # TODO: don't hard-code cftransport user
+      "transport_user" -> { "ENT-4610" } string => "cftransport";
+      "transport_home"
+        string => "$(cfengine_enterprise_federation:config.federation_dir)/cftransport";
+
+    config_present::
+      "data" data => readjson( $(path) );
+
   classes:
     enterprise_edition.(policy_server|am_policy_hub)::
-      "enabled"
-        expression => fileexists( $(path) ),
+      "config_present"
+        expression => fileexists( $(path) );
```

Comment:
TODO replace fileexists() with data validation funciton (so that we don't try to read a busted json file)
Karl is still working on it.